git-hub

```diff
@@ -757,20 +757,7 @@ class CloneCmd (object):
 
 	@classmethod
 	def run(cls, parser, args):
-		upstream = urlparse.urlsplit(args.repository).path
-		if upstream.endswith('.git'): upstream = upstream[:-4]
-		if '/' in upstream:
-			(owner, proj) = upstream.split('/')[-2:]
-			upstream = '{}/{}'.format(owner, proj)
-			for repo in req.get('/user/repos'):
-				if repo['name'] == proj and repo['fork']:
-					break
-			else: # Fork not found
-				infof('Forking {} to {}/{}', upstream,
-					config.username, proj)
-				repo = req.post('/repos/' + upstream + '/forks')
-			repo = req.get('/repos/' + repo['full_name'])
-		else: # no '/' in upstream
+		def setOwnRepoParams(upstream):
```

Comment:
Please follow the same style of the project (snake_case_functions :).

Also, I don't see why these functions should be nested, as the class only purpose is to run this command. I think it would be better to convert them in `@classmethod`s.